packages/visualizations/src/components/Map/WebGl/MapRender.svelte

```diff
@@ -125,8 +146,10 @@
         resizer = new ResizeObserver(
             debounce(() => {
                 map.resize();
-                if (mapReady && bbox) {
-                    fitMapToBbox(bbox);
+                if (mapReady && viewBox && navigationMaps) {
```

Comment:
It's a bit complicated between `bbox` and `viewBox` now I think. At some point I documented in the types so that the difference is clear, and I think we shouldn't do implicit logic here, and rather follow the documentation:
```
display the world. */
    /** Maximum boundaries of the map, outside of which the user cannot zoom/move */
    bbox?: BBox | undefined;
    /** Position of the map when rendering.
     * If undefined, will default, in order to:
     * - bbox if specified
     * - Fit the content if the source is GeoJSON
     * - The world
     */
    viewBox?: BBox | undefined;
```

So, it's up to the developer (or in our ase, the Studio) to pass the right stuff: for us, it's either:
- no viewBox, and a bbox (vtiles) or none (geojson)
- a viewBox, and no bbox (navigable maps)

You could imagine a situation where you'd pass the bbox of a country, and a smaller maps to zoom to specific territories inside it. Either way, it would be predictable. What do you think?

Reply:
Also it would probably simplify a lot of code there, e.g. `setViewBox` and `setMapBounds` will both only rely to respectively `viewBox` and `bbox`, which is easier to follow

Reply:
Indeed, I went a bit fast and I’ve missed this logic and thought we would like to only rely on viewBox with navigable maps.

I tried to fix this, so if there is a bbox it will only set the max bounds of the map and as you well described it in the documentation, the viewBox will take in this order custom values or bbox or fit content in GeoJson or the world bbox. It is responsible of only the displayed view location on the map. Hope I haven’t missed something but it seemed that in storybook it works as expected.